### ISSUE TITLE
'fallback to default color' may fail (depending on TERM setting?)

### ISSUE THREAD

**Issue by eserte** [garu/Data-Printer#128]
On some of my smokers I see the following failure (_008, _009):
```
#   Failed test 'fallback to default color'
#   at t/001-object.t line 164.
#          got: 'x'
#     expected: '[0;38;2mx[0m'
# Looks like you failed 1 test of 124.
t/001-object.t ........... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/124 subtests 
```
It seems to be reproducible by setting TERM to something like "dumb".

**Comment by garu** [comment]
Hi @eserte! Thanks for reporting - I tried setting it to "dumb" but I still can't reproduce this. It's a bit tricky having default colors that can be disabled by the user, but then again that's what tests are for and I'm glad we caught this :)

I've released _010 with some extra debugging info when that test fails, hopefully it'll be enough for us to pinpoint what went wrong.

Thanks again!

**Comment by garu** [comment]
@eserte Found it! There was a typo on the test. It's fixed now on 0.99_011, so hopefully we won't see any more issues \o/

Thanks again for all the help!
